samples/sensor/mcux_lpcmp/src/main.c

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright NXP 2024
```

Comment:
This file looks like it is based off `samples/sensor/mcux_acmp/src/main.c`, yet copyright have not been retained.

Reply:
added